.github/workflows/action.yml

```diff
@@ -20,6 +20,8 @@ jobs:
       - uses: julia-actions/julia-runtest@latest
 
   test-moonshot:
+    env:
+      CUDA_VISIBLE_DEVICES: 1
```

Comment:
Why setting this value? (I am curious :smiley_cat: )

Reply:
This makes the tests on moonshot run on `device=1` to avoid a clash with most users and #110